### PR TITLE
3.0 FIPS algs: remove the compliance column.

### PR DESCRIPTION
**Pull request by paulidale**
Removing the compliance column.
Also cleaning up some other notes which are no longer relevant because the word has been done or it isn't going to be done.

### CODE REVIEW

File: docs/OpenSSL300Design.md [hunk 1]
```diff
@@ -3082,20 +2988,6 @@ All comply with <a href="https://csrc.nist.gov/publications/detail/sp/800-90a/re
    </td>
    <td><a href="https://csrc.nist.gov/publications/detail/sp/800-90a/rev-1/final">SP 800-90A</a>
    </td>
-   <td>✓
-   </td>
-  </tr>
-  <tr>
-   <td>DRBG
```
Comment (mattcaswell) [inline docs/OpenSSL300Design.md:3089]: Why are we deleting this DRBG entry?

File: docs/OpenSSL300Design.md [hunk 2]
```diff
@@ -2880,10 +2858,6 @@ Security Policy statement regarding the <a href="https://csrc.nist.gov/publicati
    </td>
    <td><a href="https://csrc.nist.gov/publications/detail/sp/800-38d/final">SP 800-38D</a>
    </td>
-   <td>✓
-   </td>
-   <td>
-   </td>
```
Comment (mattcaswell) [inline docs/OpenSSL300Design.md:2886]: Why delete the extra columns here (and other similar removals)?

File: docs/OpenSSL300Design.md (hunk 1 above)
Comment (paulidale) [reply docs/OpenSSL300Design.md:3089]: This is the entropy source.  We're doing this outside of the FIPS boundary.

File: docs/OpenSSL300Design.md (hunk 2 above)
Comment (paulidale) [reply docs/OpenSSL300Design.md:2886]: The extra columns were inconsistently present or not.  I went with not.

**Review by mattcaswell — APPROVED** [review]

**Comment by paulidale** [comment]
Merged, thanks.

**Pull request closed without merging**